### ISSUE TITLE
Set alt text for images

### ISSUE THREAD

**Issue by JonEsparaz** [themeetinghouse/web#439]
There's lots of images with alt text "TBD" (definitely on the teaching page, likely on other pages too).

We should have something that's actually accessible.

**Comment by GeorgeBellTMH** [comment]
Can you guys go through and find any TBD's and replace with proper descriptions?

**Comment by JonEsparaz** [comment]
FYI this is mostly in the TypeScript files, so the alt text needs to be determined semi-dynamically. Probably something for a dev to take care of.

**Comment by deeinstereo** [comment]
Can I edit this using visual studio code?  I checked the teaching page now and I've already added all the alt text that I can see.

**Comment by GeorgeBellTMH** [comment]
@lucastbelem...suggest we add a new field: thumbnailDescription which would be a description of the thumbnail...ie. Thumbnail of Bruxy teaching, with the words "Q&Eh Sunday"

Someone would have to go back and fill in all this detail...

**Comment by lucastbelem** [comment]
@GeorgeBellTMH I'm fine with this, George. Would love to take stuff like this off your hands. Do you have to add the field so that we can start adding the descriptions?

**Comment by GeorgeBellTMH** [comment]
This is now in the admin...
